sklearn/metrics/regression.py

```diff
@@ -573,3 +574,36 @@ def r2_score(y_true, y_pred, sample_weight=None,
         avg_weights = multioutput
 
     return np.average(output_scores, weights=avg_weights)
+
+
+def max_error(y_true, y_pred):
+    """
+    max_error metric calculates the maximum residual error.
+
```

Comment:
``Read more in the :ref:`User Guide <...>`.``

Reply:
updated.